ames_qsar/gethlenergy.py

```diff
@@ -0,0 +1,57 @@
+#gethlenergy will take a filepath and a file name
+#and return the HOMO and LUMO energies.
+
+#this will also need a test
+import os
+
+def gethlenergy(dir_path, filename):
+
+    #initilalize the error codes as ok
+    termFlag = 0
+    errorFlag = 0
+    a = 0
+    b = 0
+    #initialize a holding variable
+    prevLine = []
+    os.chdir(dir_path)
+
+    with open(filename)as searchfile:
+
+        #These fences indicate the where the loop ends
+        #************************
+        for line in searchfile:
+            #error checking
+            #the normal termination flag should turn to 1
+            if 'Normal termination of Gaussian 09' in line:
+                termFlag = 1
+            if 'slurmstepd: error:' in line:
+                errorFlag = 1
+            #***************************************************************
+
+            left,sep,right=line.partition('Alpha virt. eigenvalues --')

```

Comment:
whitespace 'left,sep,right'

Reply:
if 'Alpha virt. eigenvalues --' in line and 'Alpha  occ. eigenvalues --' in prev_line: